lib/players/human.rb

```diff
@@ -0,0 +1,12 @@
+require 'pry'
+
+class Human < Player
+
+  def move(input)
+    input = " "
+    input = gets.chomp
+    input
```

Comment:
You don't need to store your return value in a local variable; just utilize the implicit return value of the expression.
```
def move(input)
  gets.chomp
end
```